apps/tup-ui/src/pages/Projects/Projects.module.css

```
@@ -1,5 +1,16 @@
-.project-header {
-    font-size: 2.6rem;
-    border-bottom: var(--global-border-width--normal) solid var(--global-color-primary--dark);
-    padding-bottom: 5px;
+.project-section {
+    display: grid;
+}
+
+.project-section > div > header {
+    padding-top: var(--global-space--section-top);
+    padding-left: var(--global-space--section-left);
+}
+
+.project-section nav {
+    margin-top: calc( -1 * var(--global-space--section-top));
```

Reasonable solution, given that we have no upstream solution to manage multiple layouts.

But, the use of `--global-space--section-top` is misleading[^1] and creates a minor, dormant space bug; because the space being removed is [not 15px via `--global-space--section-top`](https://github.com/TACC/tup-ui/blob/1cf5b33/libs/core-components/src/lib/SectionHeader/SectionHeader.module.css#L10).[^2]

```suggestion
    margin-top: calc( -1 * 15px); /* remove `<SectionHeader> `margin-bottom` */
```

[^1]: You could add a new global variable, e.g. `--global-space--header-bottom`, in [`/apps/tup-ui/src/main.global.css`](https://github.com/TACC/tup-ui/blob/f051aad/apps/tup-ui/src/main.global.css).
[^2]: Design just happened to use approx. 15px in another spot.